apiserver/common/crossmodel/crossmodel.go

```diff
@@ -37,7 +37,7 @@ func PublishRelationChange(backend Backend, relationTag names.Tag, change params
 
 	// Update the relation suspended status.
 	currentStatus := rel.Suspended()
-	if change.Suspended != nil && currentStatus != *change.Suspended {
+	if !dyingOrDead && change.Suspended != nil && currentStatus != *change.Suspended {
```

Comment:
does a relation have to be unsuspended before it can be removed? just wondering why we would prevent this on Dying

Reply:
Dying takes precedence. If a relation were suspended for example, and then someone removes it, we just want it to die. There's no need for any further processing of suspended status.